test/jdk/java/lang/System/SecurityManagerWarnings.java

```diff
@@ -31,6 +31,8 @@
 import jdk.test.lib.process.OutputAnalyzer;
 import jdk.test.lib.process.ProcessTools;
 
+import java.io.ByteArrayOutputStream;
+import java.io.PrintStream;
 import java.security.Permission;
 
 public class SecurityManagerWarnings {
```

Comment:
This should probably have 8268349 in the @bug line.

Reply:
OK.